workflow/rocoto/gfs_tasks.py

```diff
@@ -1175,9 +1175,20 @@ def wavepostbndpnt(self):
         return task
 
     def wavepostbndpntbll(self):
+
+        # The wavepostbndpntbll job runs on forecast hours up to FHMAX_WAV_IBP
+        wave_cfg = self._configs['wave']
+        last_fhr = Tasks._get_forecast_hours(self.cdump,
+                                             wave_cfg,
+                                             component='atmos')[-1]
+
+        fhmax_wav_ibp = wave_cfg['FHMAX_WAV_IBP']
+        if last_fhr > fhmax_wav_ibp:
+            last_fhr = fhmax_wav_ibp
+
         deps = []
         atmos_hist_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_HISTORY_TMPL"])
-        data = f'{atmos_hist_path}/{self.cdump}.t@Hz.atm.logf180.txt'
+        data = f'{atmos_hist_path}/{self.cdump}.t@Hz.atm.logf{last_fhr:03d}.txt'
```

Comment:
@WalterKolczynski-NOAA `FHMAX_GFS` and `FHMAX_WAV_IBP` can differ if `FHMAX_GFS` is greater than 180 hours.

Reply:
@JessicaMeixner-NOAA It seems like `FHMAX_WAV_IBP` should default to `FHMAX_WAV` instead of `FHMAX_GFS` in
https://github.com/NOAA-EMC/global-workflow/blob/e7909af8d9e1f34140388a3f8556d8e582c58fe5/parm/config/gfs/config.wave#L100-L105
And thus change to
```bash
if [[ "${RUN}" == "gfs" ]]; then
  export FHMAX_WAV=${FHMAX_WAV_GFS}
fi
export WAVHINDH=0
export FHMAX_WAV_IBP=180
if (( FHMAX_WAV < FHMAX_WAV_IBP )); then export FHMAX_WAV_IBP=${FHMAX_WAV} ; fi
```
Do you think that is correct?

Reply:
@DavidHuber-NOAA nice catch, yes I do think this is correct.